### PR TITLE
cpu: aarch64: fix bug in make softmax ops use stateless ACL interface

### PR DESCRIPTION
**Pull request by taoye9**
# Description

A patch to fix bug in recently merged PR https://github.com/oneapi-src/oneDNN/pull/2136. 

# Checklist

## General

- [x] Do all unit and benchdnn tests (`make test` and `make test_benchdnn_*`) pass locally for each commit?
- [x] Have you formatted the code using clang-format?

### CODE REVIEW

**Comment by Sqvid** [comment]
Remove the change-id

**Review by Radu2k — APPROVED** [review]
LGTM.

File: src/cpu/aarch64/acl_softmax.cpp [hunk 1]
```diff
@@ -111,8 +107,8 @@ status_t acl_softmax_fwd_t::pd_t::init(engine_t *engine) {
         // A rough empirical heuristic, see comment above
         // The only difference here is that ACL does a reorder, and so
         // is considerably better
-        double acl_ref_performance_diff = calculate_performance_diff(
-                outer_size_, axis_size_, threads, -0.01);
+        double acl_ref_performance_diff
+                = calculate_performance_diff(-0.01 * inner_size_);
```
Comment (mgouicem) [inline src/cpu/aarch64/acl_softmax.cpp:111]: IIUC, this is the only impactful change (change from `-0.01` to `-0.01 * inner_size_`), is that correct?

**Review by mgouicem — APPROVED** [review]

File: src/cpu/aarch64/acl_softmax.cpp (hunk 1 above)
Comment (taoye9) [reply src/cpu/aarch64/acl_softmax.cpp:111]: Yes, it's the only line of code.

**Comment by taoye9** [comment]
ci fails due to other issues: https://github.com/oneapi-src/oneDNN/pull/2176

**Comment by taoye9** [comment]
hi, @mgouicem this pr passes ci tests, could you please merge it at your convenience?

**Pull request merged**